package.json

```diff
@@ -11,10 +11,11 @@
   },
   "main": "index.js",
   "scripts": {
-    "prepare": "npm prune && npm run build",
-    "build": "rm -rf lib && babel src --out-dir lib -s > /dev/null",
+    "prepare": "pwd && rm -rf ./lib",
+    "postinstall": "npm run build",
```

Comment:
I'm a bit concerned with moving transpilation step to the user machine where there could be problems with conflicting version and operating systems. I rather package everything on my machine where I know everything is working and push it with package.

Reply:
I think it'll be safer if we move `build` step back into `prepare`

Reply:
@stsvilik I struggled with `build` being run `npm prepare` when I used `wdio-docker-service` as a URL package, because according to the docs [0] this runs before `npm install`

I would create an additional task for `prePublishOnly` [1]

[0] `prepare: Run both BEFORE the package is packed and published, on local npm install without any arguments, and when installing git dependencies` https://docs.npmjs.com/misc/scripts
[1] `A new event, prepublishOnly has been added as a transitional strategy to allow users to avoid the confusing behavior of existing npm versions and only run on npm publish (for instance, running the tests one last time to ensure they’re in good shape).`